tests/unit/array/mixins/test_empty.py

```diff
@@ -24,14 +24,15 @@
     ],
 )
 def test_empty_non_zero(da_cls, config, start_storage):
-
     # Assert .empty provides a da with 0 docs
     if config:
         da = da_cls.empty(config=config)
     else:
         da = da_cls.empty()
 
     assert len(da) == 0
+    if da_cls == DocumentArrayAnnlite:
+        da.close()
```

Comment:
remove this, this is not what we are testing, we cannot test custom behavior like this. WE AIM FOR SAME interface

Reply:
Do you think we need to refactor the annlite-based test? We need to manually close it before another test starts. Otherwise, the file lock will block us.

Reply:
I do not like imposing the `close` API, this needs to be fix on ANNLite backend IMO